app/views/foreman_tasks/tasks/dashboard/_tasks_status.html.erb

```diff
@@ -4,12 +4,14 @@
     <th><%= _("State") %></th>
     <th><%= _("Result") %></th>
     <th><%= _("No. of Tasks") %></th>
+    <th><%= _("Last start time") %></th>
   </tr>
   <% ForemanTasks::Task::Summarizer.new.summarize_by_status.each do |result| %>
     <tr class="<%= ForemanTasks::Task::StatusExplicator.new.is_erroneous(result) ? 'text-danger' : '' %>">
       <td><%= result.state %></td>
       <td><%= result.result %></td>
       <td><%= link_to result.count, main_app.foreman_tasks_tasks_path(:search => "state=#{result.state}&result=#{result.result}") %></td>
+      <td><%= _('%s ago') % time_ago_in_words(result.started_at) %></td>
```

Comment:
While testing the PR, I've noticed that I have a case where started_at was nil. Although it might be a corner case, in this case, I would rather be safe than sorry and check the `started_at` presence, similarly to https://github.com/theforeman/foreman-tasks/pull/273